tests/integration/test_script_runner.py

```diff
@@ -712,9 +712,10 @@ def _execute_integration_test(integration_test_fixture, tmp_path):
     workdir = os.getcwd()
     try:
         base_dir = file_relative_path(__file__, "../../")
-        os.chdir(tmp_path)
+        os.chdir(base_dir)
         # Ensure GE is installed in our environment
         execute_shell_command("pip install .")
+        os.chdir(tmp_path)
```

Comment:
Need to explicitly change to the `base_dir` (ie. where `great_expectations/` is) in order to do the installation before switching back to our current `tmp_path`